@here/harp-examples/src/threejs_add-simple-object.ts

```diff
@@ -75,36 +75,22 @@ export namespace ThreejsAddSimpleObject {
             }
             // snippet:harp_gl_threejs_add_simple_object_1.ts
             // Get the position of the mouse in world space.
-            const worldPositionAtMouse = mapView.getWorldPositionAt(event.pageX, event.pageY);
-            if (worldPositionAtMouse === null) {
+            const geoPosition = mapView.getGeoCoordinatesAt(event.pageX, event.pageY);
+            if (geoPosition === null) {
                 return;
             }
             // end:harp_gl_threejs_add_simple_object_1.ts
 
             // snippet:harp_gl_threejs_add_simple_object_2.ts
             const cube = createPinkCube();
-            mapView.scene.add(cube);
+            cube.geoPosition = geoPosition;
+            mapView.userMapObjects.add(cube);
             // end:harp_gl_threejs_add_simple_object_2.ts
 
-            const onRender = () => {
```

Comment:
Please adjust the documentation above

Reply:
you mean the line "            // Get the position of the mouse in world space." ? or is there more to update?